addons/project/controllers/portal.py

```diff
@@ -313,97 +313,64 @@ def _task_get_page_view_values(self, task, access_token, **kwargs):
         return values
 
     def _task_get_searchbar_groupby(self, milestones_allowed, project=False):
         values = {
-            'none': {'input': 'none', 'label': _('None'), 'order': 1},
-            'stage': {'input': 'stage', 'label': _('Stage'), 'order': 4},
-            'status': {'input': 'status', 'label': _('Status'), 'order': 5},
-            'priority': {'input': 'priority', 'label': _('Priority'), 'order': 7},
-            'customer': {'input': 'customer', 'label': _('Customer'), 'order': 10},
+            'none': {'label': _('None'), 'sequence': 10},
+            'stage_id': {'label': _('Stage'), 'sequence': 20},
+            'state': {'label': _('Status'), 'sequence': 40},
+            'priority': {'label': _('Priority'), 'sequence': 60},
+            'partner_id': {'label': _('Customer'), 'sequence': 70},
         }
         if not project:
-            values['project'] = {'input': 'project', 'label': _('Project'), 'order': 2}
+            values['project_id'] = {'label': _('Project'), 'sequence': 30}
         if milestones_allowed:
-            values['milestone'] = {'input': 'milestone', 'label': _('Milestone'), 'order': 6}
-        return dict(sorted(values.items(), key=lambda item: item[1]["order"]))
-
-    def _task_get_groupby_mapping(self):
-        return {
-            'project': 'project_id',
-            'stage': 'stage_id',
-            'customer': 'partner_id',
-            'milestone': 'milestone_id',
-            'priority': 'priority',
-            'status': 'state',
-        }
-
-    def _task_get_order(self, order, groupby):
-        groupby_mapping = self._task_get_groupby_mapping()
-        field_name = groupby_mapping.get(groupby, '')
-        if not field_name:
-            return order
-        return '%s, %s' % (field_name, order)
+            values['milestone_id'] = {'label': _('Milestone'), 'sequence': 50}
+        return values
 
     def _task_get_searchbar_inputs(self, milestones_allowed, project=False):
         values = {
-            'all': {'input': 'all', 'label': _('Search in All'), 'order': 1},
-            'content': {'input': 'content', 'label': Markup(_('Search <span class="nolabel"> (in Content)</span>')), 'order': 1},
-            'ref': {'input': 'ref', 'label': _('Search in Ref'), 'order': 1},
-            'users': {'input': 'users', 'label': _('Search in Assignees'), 'order': 3},
-            'stage': {'input': 'stage', 'label': _('Search in Stages'), 'order': 4},
-            'status': {'input': 'status', 'label': _('Search in Status'), 'order': 5},
-            'priority': {'input': 'priority', 'label': _('Search in Priority'), 'order': 7},
-            'customer': {'input': 'customer', 'label': _('Search in Customer'), 'order': 10},
-            'message': {'input': 'message', 'label': _('Search in Messages'), 'order': 11},
+            'name': {'input': 'name', 'label': Markup(_('Search<span class="nolabel"> Tasks</span>')), 'sequence': 10},
```

Comment:
Hello @odoo/rd-security 

Can we have an override?

We just move the Markup function. 🙂 

Thanks in advance!

Have a nice afternoon! 🙂

Reply:
Why is there that weird span on one word? if you rearrange you could do something nicer like
```suggestion
            'name': {'input': 'name', 'label': Markup('<span class="nolabel">%s</span>') % _('Search Tasks'), 'sequence': 10},
```
We try to avoid having translated html

Reply:
If you really want to have this span in the middle would be something like
```py
_("Search %(left)s Tasks %(right)s",
    left=Markup('<span class="nolabel">'),
    right=Markup('</span>'))
```

Reply:
@mart-e hum indeed, I will let @MissingNoShiny check that.

Thanks for your review. 😘 

Have a nice afternoon!